src/auth/auth.module.ts

```diff
@@ -0,0 +1,20 @@
+import { Module } from '@nestjs/common';
+import { AuthService } from './auth.service';
+import { JwtStrategy } from './jwt.strategy';
+import { UsersModule } from '../users/users.module';
+import { PassportModule } from '@nestjs/passport';
+import { JwtModule } from '@nestjs/jwt';
+
+@Module({
+  imports: [
+    UsersModule,
+    PassportModule,
+    JwtModule.register({
+      secret: 'secret', // TODO update
```

Comment:
TBH, IMO such comments (todo) doesn't work mostly in real work. Most people just forget about them or othera devs aren't aware when doing their own task is this a good moment to do something with it or not